pipeline_config/default.json

```diff
@@ -1,14 +1,24 @@
 {
-   "data":"data",
-   "export":{
-      "era5":[
-         {
-            "variable":"total_precipitation"
-        },
-      ],"vhi":[
-          {
-              "years":["1981"]
-          }
-      ]
-   }
+    "data": "data",
+    "export": {
+        "vhi": [
+            {"years": [2010,
```

Comment:
@gabrieltseng Just a subset of the years for the default?

Reply:
(To be revisited in another PR)